src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -220,15 +280,19 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     }
 
     initializeDonationDonorTypeInFormState(donorApiName) {
-        if (donorApiName === 'Account') {
-            this.updateFormState({
-                [DATA_IMPORT_DONATION_DONOR_FIELD]: DONATION_DONOR_TYPE_ENUM.ACCOUNT1
-            });
-        }
-        if (donorApiName === 'Contact') {
-            this.updateFormState({
-                [DATA_IMPORT_DONATION_DONOR_FIELD]: DONATION_DONOR_TYPE_ENUM.CONTACT1
-            });
+        const updates = new Map();
+        updates.set(DATA_IMPORT_DONATION_DONOR_FIELD, donorApiName);
+        this.updateFormState(updates);
+    }
+
+    getDonationDonorTypeFor(donorApiName) {
```

Comment:
Are we still calling this somewhere by chance?

Reply:
Nope, doesn't look like it.  Let's remove it in the next WI that modifies this class.